src/test/java/com/epam/qavn/webdriverio/formPage/SubmitFormTests.java

```diff
@@ -21,8 +21,9 @@ public void activeFormTest() {
                 .tapFormsMenu()
                 .fillForm("This is text", true, FormPage.FormDropdown.APPIUM)
                 .tapButtonActive()
-                .isActiveMessageDisplayed();
+                .isActiveMessagePopupDisplayed();
         Assert.assertTrue(messageIsDisplayed, "Active message is not displayed");
+        formPage.tapButtonOk();
```

Comment:
move this action to After.
please refer to LoginTests class from line 31 to line 40

Reply:
Updated